.github/CODEOWNERS

```diff
@@ -32,6 +32,7 @@ system/ @Ferroin @tkatsoulas
 tests/ @Ferroin @vkalintiris @tkatsoulas
 web/ @thiagoftsm @vkalintiris
 web/gui/ @novykh
+logsmanagement/ @Dim-P
```

Comment:
Hello @Dim-P,

Please, also add the collector team as code owners here.

Best regards!

Reply:
I can see that's only you @thiagoftsm ! I added you.